writing/referencing.md

```diff
@@ -0,0 +1,15 @@
+# 1 Referencing
+
+- [1 Referencing](#1-referencing)
+  - [1.1 Format](#11-format)
+  - [1.2 Example](#12-example)
+
+## 1.1 Format
+
+Meant to be Harvard referncing. I'm not 100% certain this is right, will have to check.
+
+[Author]. ([Year]) [Article Title], [Publication Name]. [online] Available at: <[url]> [Accessed: [Access Date]]
```

Comment:
This is probably not completely correct. Need to read through the guidelines and information. There are also slight variations for different scenarios (Different author types, materials, etc.)